### ISSUE TITLE
Remove 'q_a_pairs.csv' and the reference to it, replace it with the answerable_questions API call

### ISSUE THREAD

**Issue by richagadgil** [calpoly-csai/api#132]
https://github.com/calpoly-csai/api/blob/b5bde1447021700c0e1dbc4907b867f391bf3c18/QA.py#L190

Remove `q_a_pairs.csv` and the reference to it, replace it with the answerable_questions API call.
See #131 to see how answerable_questions api is used to populate dataframe.

**Comment by mfekadu** [comment]
@richagadgil  does #131  take care of this issue?

**Comment by richagadgil** [comment]
No, this issue is requesting @cameron-toy to replace his use of the q_a_pairs.csv with an API call to answerable_questions when possible. He can use my code as a reference because it does the same thing :)

**Comment by richagadgil** [comment]
Then hopefully we can remove q_a_pairs.csv from the api repo

**Comment by cameron-toy** [comment]
@richagadgil This has been resolved right?
